### ISSUE TITLE
When zoomed in, all REPL errors show the "scroll down" arrow in definitions

### ISSUE THREAD

**Issue by jpolitz** [brownplt/pyret-lang#418]
To reproduce:

1.  Go to a larger zoom level than 100%
2.  Type a correct program into definitions, run it
3.  Cause some error in interactions, like `x - y` with neither bound
4.  Hover over the error box --- the down arrow will appear

This doesn't happen at 100%.  Noticed in Chrome 39.0 on Ubuntu

![image](https://cloud.githubusercontent.com/assets/119891/5620405/c4d27c72-94f9-11e4-8c96-a81366ccd214.png)

**Comment by schanzer** [comment]
Given how different error messages are now than they were in 2015 - this seems like it should be closed.

**Comment by Emmay** [comment]
Agreed with Emmanuel- I can't reproduce this in chrome at any zoom level. Is this closeable?
